internal/tmmsgs/en_config_descriptions.go

```diff
@@ -44,6 +45,7 @@ var (
 	ConfigConfirmationsNotificationsQueueLength = ffc("config.confirmations.notificationQueueLength", "Internal queue length for notifying the confirmations manager of new transactions/events", i18n.IntType)
 	ConfigConfirmationsRequired                 = ffc("config.confirmations.required", "Number of confirmations required to consider a transaction/event final", i18n.IntType)
 	ConfigConfirmationsStaleReceiptTimeout      = ffc("config.confirmations.staleReceiptTimeout", "Duration after which to force a receipt check for a pending transaction", i18n.TimeDurationType)
+	ConfigConfirmationsReceiptWorkers           = ffc("config.confirmations.receiptWorkers", "Number of workers to use to query in parallel for receipts", i18n.IntType)
 
 	ConfigTransactionsMaxHistoryCount = ffc("config.transactions.maxHistoryCount", "The number of historical status updates to retain in the operation", i18n.IntType)
 
```

Comment:
Just below here is a `DeprecatedConfigTransactionsNonceStateTimeout`, which I believe has been un-deprecated, and further down is a `ConfigTXHandlerNonceStateTimeout` among the "simple" handler configs, which I think has been deprecated.

Reply:
Sorry for leaving you to work this out 🤦 - yes you're 100% right